packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-editor.tsx

```diff
@@ -21,20 +23,41 @@ import { changeEditorValue } from '../../../modules/pipeline-builder/text-editor
 import type { PipelineParserError } from '../../../modules/pipeline-builder/pipeline-parser/utils';
 
 const containerStyles = css({
+  position: 'relative',
   display: 'flex',
   flexDirection: 'column',
   height: '100%',
-  justifyContent: 'space-between',
+  backgroundColor: palette.gray.light3,
+  paddingTop: spacing[3],
+  paddingBottom: spacing[2],
```

Comment:
In the designs editor takes all the space on the left, adjusting color and paddings to make it look like that